src/components/images_uploading/UploadImageFile.js

```diff
@@ -14,7 +13,9 @@ class UploadImageFile extends Component {
   };
 
   fileUploadHandler = event => {
-    event.preventDefault();

```

Comment:
The only way I found to refresh the page once we create a new apartment lol
I know this is definitely not the best way to do it, but at least it gets the work done XD
Will see if I can find a better way later :)